### PR TITLE
Array operations in JAVA lang

### PR DESCRIPTION
**Pull request by unon4all**
It should work this time.

### CODE REVIEW

**Comment by unon4all** [comment]
Few things need to be updated, just some Print statements … I will update them as soon as I finish my college assignments and practicals.

File: Codes/ArraysJava.txt [hunk 1]
```diff
@@ -0,0 +1,557 @@
...
+            //Enter your choice
+            System.out.println("Enter your choice: ");
+            X = in.nextInt();
+
+            //Switch case
+
+            switch (X) 
+            {
+                case 1:
+                    //Append at the Starting
```
Comment (Saurabhsharma884) [inline Codes/ArraysJava.txt:96]: Make sure previously inserted elements remain unchanged
This code will overwrite first element of array , please correct this

File: Codes/ArraysJava.txt [hunk 2]
```diff
@@ -0,0 +1,557 @@
...
+        arr[index]=number;
+        return arr;
+    }
+
+    //MAXIMUM OF AN ARRAY
+    public static int maxElement(int[] arr)
+    {   
+        int max=arr[0];
+        for (int i = 0; i < arr.length; i++) {
+            if(arr[i]>0)
```
Comment (Saurabhsharma884) [inline Codes/ArraysJava.txt:348]: if condition is wrong, please recheck

File: Codes/ArraysJava.txt [hunk 3]
```diff
@@ -0,0 +1,557 @@
...
+                min=arr[i];
+        }
+        return min;
+    }
+
+    //SUM OF ALL ELEMENT 
+    public static int sumArray(int[] arr) 
+    {
+        int sum=0;
+        int temp=0;
```
Comment (Saurabhsharma884) [inline Codes/ArraysJava.txt:368]: No need of using temp variable

File: Codes/ArraysJava.txt [hunk 4]
```diff
@@ -0,0 +1,557 @@
...
+            mergeArr[pos]=sec_Array[i];
+            pos++;
+        }
+
+        return mergeArr;
+    }
+
+    //UNION OF TWO ARRAYS
+
+    public static void unionArray(int[] arr) {
```
Comment (Saurabhsharma884) [inline Codes/ArraysJava.txt:497]: Don't use HashSet,Set or Map for Union. Try to do it with Arrays only

**Comment by Bhuvnesh875** [comment]
hey @unon4all , i see there are some changes that you need to do in your code, i appreciate the time and effort you put in this PR but there is a  major update rolling out today so pls make the necessary changes according to the new contributing guidelines and open a PR again if you can, thank you!

**Pull request closed without merging**